tests/candev/main.c

```diff
@@ -229,10 +229,11 @@ if (IS_ACTIVE(CONFIG_USE_LOOPBACK_MODE)) {
 #if defined(MODULE_MCP2515)
         filter[0].target_mailbox = 0;   /* messages with CAN ID 0x001 will be received in mailbox 0 */
 #endif
-        filter[1].can_mask = 0x7FF;
-        filter[1].can_id = 0x002;
+        filter[1].can_mask = 0x9FFFFFFF;
+        filter[1].can_id = 0x8C11FFFF;
```

Comment:
How are those values generated?

Reply:
Just examples with the "extended frame" bit (MSB) set

Reply:
Which examples?

Reply:
An example of an extended CAN filter usage.

Reply:
This test serves as an example for extended CAN filter usage.
We should at least have a comment here as to how a user arrives at those numbers.

Ideally we would use defines to construct this `can_id` and `can_mask`.